iOS/DuckDuckGo/MainViewController.swift

```diff
@@ -1624,6 +1626,13 @@ class MainViewController: UIViewController {
             }
             .store(in: &vpnCancellables)
 
+        NotificationCenter.default.publisher(for: .expiredRefreshTokenDetected)
+            .receive(on: DispatchQueue.main)
+            .sink { [weak self] notification in
+                self?.onExpiredRefreshTokenDetected(notification)
+            }
+            .store(in: &vpnCancellables)
```

Comment:
Not sure this is the right place to put it since it is `subscribeToNetworkProtectionEvents`

Reply:
entitlementsDidChange and accountDidSignOut are also in the function, and they are subscription and authentication notifications, I could create a new subscription only function, but I don't think really matters